checkout_deps.sh

```diff
@@ -15,6 +15,10 @@
 # limitations under the License.
 
 # hiredis version 0.13.3
+if [ -d "third_party" ]; then
+  echo "Directory third_party exists!"
```

Comment:
maybe add a helpful note, something like "Please remove this directory before running build_deps.sh"